### PR TITLE
Release 6.0.0-alpha.1

### PR DESCRIPTION
**Pull request by r0mant**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 6.0.0-alpha.1
+
+This release of Teleport introduces Database Access with PostgreSQL support.
```
Comment (russjones) [inline CHANGELOG.md:5]: Can you add a block to this about it being a pre-release.

**Review by russjones — APPROVED** [review]
Other than the pre-release comment, lgtm.

File: CHANGELOG.md (hunk 1 above)
Comment (r0mant) [reply CHANGELOG.md:5]: Done.

**Comment by r0mant** [comment]
@benarent @webvictim @klizhentas Looks like need one of your folks approval as well as code owners for this file.

**Review by klizhentas — APPROVED** [review]

**Review by a-palchikov — APPROVED** [review]

**Pull request merged**